CMakePresets.json

```diff
@@ -0,0 +1,216 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 21,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "nix-base",
+      "hidden": true,
+      "generator": "Ninja",
+      "cacheVariables": {
+        "CMAKE_CXX_COMPILER_LAUNCHER": "ccache",
+        "CMAKE_FIND_FRAMEWORK": "LAST",
+        "CMAKE_FIND_PACKAGE_PREFER_CONFIG": "ON",
+        "CMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY": "OFF",
+        "CMAKE_FIND_USE_PACKAGE_REGISTRY": "OFF",
+        "CMAKE_OSX_SYSROOT": "",
+        "VAST_ENABLE_DEVELOPER_MODE": "ON",
+        "VAST_ENABLE_LSVAST": "ON",
+        "VAST_ENABLE_MDX_REGENERATE": "ON",
+        "VAST_PLUGINS": "${sourceDir}/plugins/broker;${sourceDir}/plugins/pcap;${sourceDir}/plugins/sigma;${sourceDir}/plugins/summarize"
```

Comment:
Is there a way to generate this dynamically based on the source tree, so that it doesn't have to be updated manually?

(Applies to all occurrences of `VAST_PLUGINS`.)

Reply:
```suggestion
        "VAST_PLUGINS": "${sourceDir}/plugins/*"
```

This already works.

Reply:
Not that I know of. At least on in this file. We could add an `VAST_ENABLE_BUNDLED_PLUGINS` option to CMakeLists.txt. @dominiklohmann will probably have an opinion on that.

Reply:
What I specified already works, we expand the list of plugins via globbing.